File: app/src/main/java/com/BlizzardArmory/warcraft/Equipment/Faction.java

package com.BlizzardArmory.warcraft.Equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Faction {

    @SerializedName("value")
    @Expose
    private Value value;
    @SerializedName("display_string")
    @Expose
    private String displayString;

    public Value getValue() {
        return value;
    }

    public void setValue(Value value) {
        this.value = value;
    }

    public String getDisplayString() {
        return displayString;
    }

    public void setDisplayString(String displayString) {
        this.displayString = displayString;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/account/Faction.kt
package com.BlizzardArmory.model.warcraft.account

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * The type Faction.
 */
data class Faction(

        @SerializedName("type")
        @Expose
        var type: String,

        @SerializedName("name")
        @Expose
        var name: String

)